RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -431,10 +434,13 @@
             </ToolBar>
         </ToolBarTray>
 
-        <controls:EmptyUIRefresh Grid.Row="1" />
+        <TextBox Grid.Row="1" TextChanged="SearchBox_OnTextChanged" VerticalContentAlignment="Center" Name="SearchBox"></TextBox>
+        <Image Grid.Row="1" Source="{StaticResource SearchImage}" HorizontalAlignment="Right" VerticalAlignment="Center" MaxHeight="16" Margin="0,0,1,0" MouseDown="SearchIcon_OnMouseDown" />
```

Comment:
I have to handle mouse down so clicking the search sends focus to the textbox; I think this is OK.